include/sysapp/switch.h

```diff
@@ -19,8 +19,82 @@
 extern "C" {
 #endif
 
-//! Argument struct for \link SYSSwitchToBrowserForViewer \endlink.
+typedef struct SysAppStandardArgs SysAppStandardArgs;
 typedef struct SysAppBrowserArgs SysAppBrowserArgs;
+typedef struct SysAppBrowserArgsWithCallback SysAppBrowserArgsWithCallback;
+typedef struct SysAppEShopArgs SysAppEShopArgs;
+typedef struct SysAppEManualArgs SysAppEManualArgs;
+
+struct SysAppStandardArgs
+{
+    void *    anchor;
```

Comment:
We do not use this struct member padded alignment anywhere else in wut, so should probably not use it here just for consistency.

Reply:
Done